.circleci/config.yml

```diff
@@ -1,3 +1,19 @@
+aliases:
+  # Uses the same regex as release-tags-job
+  release-tags-workflow: &release-tags-workflow
+    when:
+      and:
+        - matches: { pattern: "^[0-9]+\\.[0-9]+\\.[0-9]+(?!.*-SNAPSHOT).*$", value: << pipeline.git.tag >> }
+        - not:
+            equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
+  # Uses the same regex as release-tags-workflow
+  release-tags-job: &release-tags-job
```

Comment:
Hmm I wonder if we should just rename to `release-tags` since it's how we mostly name this for the other SDKs?

Reply:
Yea I named it like this to differentiate it from `release-tags-workflow`, which uses the same regex, but needs to use `when` instead of `filters`.